scss/partials/_cards.scss

```diff
@@ -0,0 +1,114 @@
+/** TABLE OF CONTENT-Cards
+ — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — /
+1 — Content Block
+  1.1 — Date
+  1.2 — Something Else
+
+/// END TABLE OF CONTENTS **/
+
+.content-block {
+  padding-right: 1.8rem;
+  position: relative;
+  width: calc(100vw - 2.8rem);
+
+  @include breakpoint(tablet){
+    //TODO: remove margin on last card
+    padding-right: 7.5rem;
+    width: calc(100vw - 8.5rem);
+  }
+
+  &__button {
+    display: none;
+    @include breakpoint(tablet) {
+      display: inline-block;
+    }
+  }
+  
+  &__date,
+  &__image {
+    align-self: center;
+    grid-row: 1 / 2;
+  }
+  
+  &__date {
+    @extend .uppercase;
+    align-items: center;
+    background-color: $secondary-blue-1;
+    color: $primary-white;
+    display: flex;
+    flex-direction: column;
+    grid-column: 1 / -1;
+    justify-content: center;
+    margin-left: -1rem;
+    height: 9rem;
+    width: 9rem;
+    
+    @include breakpoint(tablet){
+      margin-left: -2rem;
+    }
+    
+    &-month {
+      @extend .tk-aktiv-grotesk;
+      font-size: 1.4rem;
+      font-weight: 700;
+      letter-spacing: .3rem;
+      line-height: 1.6rem;
+    }
+    
+    &-day {
+      @extend .tk-degular-text;
+      font-size: 4.2rem;
+      font-weight: 600;
+      line-height: 4.6rem;
+    }
+  }
+  
+  &__image {
+    grid-column: 1 / 3;
+    width: 100%;
+    @include breakpoint(tablet){
+
+    }
+  }
+  
+  &__subtitle {
+    @extend [class^="tertiary-info"];
+    color: $secondary-green-1;
+    margin-bottom: 1rem;
+    
+    @include breakpoint(tablet){
+      margin-bottom: 1.4rem;
+    }
+  }
+  
+  &__text {
+    margin-bottom: 2.4rem;
+  }
+  
+  &__title {
+    margin-bottom: 1rem;
+    @include breakpoint(tablet){
+      margin-bottom: 1.4rem;
+    }
+    & > a {
+      text-decoration: none;
+      display: inline !important;
+    }
```

Comment:
Need guidance -> Not an elegant solution. Using the anchor class introduces `display: inline-block` which because of that the border is applied to the link as a block and not as an underline to the text.